helpers.js

```diff
@@ -0,0 +1,11 @@
+
+const calculateSongAvgRating = (favorites) => {
+  if (favorites.length === 0) return 0
+  const totalRating = favorites.reduce((sum, favorite) => {
+    sum += favorite.rating
+    return sum
+  }, 0)
+  return totalRating / favorites.length
```

Comment:
Created a helper method for calculating song average, let me know what you think

Reply:
Looks great! I think we might get feedback we should do this via a database query, but it looks cleaner than the one we had.